src/libraries/Microsoft.Extensions.Configuration.Binder/gen/ConfigurationBindingGenerator.Parser.cs

```diff
@@ -63,6 +63,8 @@ public Parser(SourceProductionContext context, KnownTypeSymbols typeSymbols, Imm
                     }
                 }
 
+                _sourceGenSpec.EmitThrowIfNullMethod = !_typeSymbols.ArgumentNullException.GetMembers("ThrowIfNull").IsEmpty;
```

Comment:
```suggestion
                _sourceGenSpec.EmitThrowIfNullMethod = _typeSymbols.ArgumentNullException?.GetMembers("ThrowIfNull").IsEmpty is false;
```

The symbol could be null; avoid a null-ref exception.